docs/doxygen/migration/shadow.txt

```diff
@@ -0,0 +1,62 @@
+/**
+@page shadow_migration Shadow
+@brief How to migrate an Shadow application from v3 to 202009.00.
+
+The Device Shadow library has been redesigned in the Long Term Support (LTS) release.  The new design has the following differences than the Version 3 of the AWS IoT Device SDK for Embedded C.
+
+- The new Device Shadow library is completely decoupled from an underlying MQTT library.  The Version 3 library depended on the same version of MQTT library in the AWS IoT Device SDK for Embedded C. <br>
+- The new library does not have any code for handling JSON files.  The Version 3 library contained code for assembling simple JSON files in the format expected by the Device Shadow service. <br>
+- The new library only provides simple convenience macros and functions for handling MQTT topic strings defined by AWS IoT Device Shadow service.  It does not contain any functionality of a MQTT stack.  The Version 3 library, on the other hand, provided a complete set of API for interacting with the service by using a MQTT stack underneath. <br>
+
+An application can continue using the Version 3 Device Shadow library if it also chooses to keep using the same version of MQTT library of the AWS IoT Device SDK for Embedded C.
+Regardless an application uses the Version 3 MQTT library or any MQTT library of its choice, the application can use the new Device Shadow library by following the call sequence diagram in the [Design document](@ref shadow_design).
```

Comment:
```suggestion
An application may use the version 3 MQTT library, version 202009.00 MQTT library, or any other MQTT library with the new Device Shadow library. To use the new Device Shadow library with any MQTT library, please see the call sequence diagram in [Design document](@ref shadow_design). 
```

Reply:
plz check [the commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1226/commits/0f6b5f0f9bdb5f6fd2ce1e55545e99e6c3563f2d).